src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -6482,6 +6866,79 @@ namespace HVACVariableRefrigerantFlow {
+                Real64 MaxSATSuppHeatDes = FinalZoneSizing(CurZoneEqNum).HeatDesTemp;
+                if (IsAutoSize) {
+                    VRFTU(VRFTUNum).MaxSATFromSuppHeatCoil = MaxSATSuppHeatDes;
+                    ReportSizingOutput(cZoneHVAC_TU_VRF,
+                                       ZoneHVAC_TU_VRF_Name,
+                                       "Design Size Maximum Supply Air Temperature from Supplemental Heater [C]",
+                                       MaxSATSuppHeatDes);
+                } else {
+                    if (VRFTU(VRFTUNum).MaxSATFromSuppHeatCoil > 0.0 && MaxSATSuppHeatDes > 0.0 && SizingDesRunThisZone) {
+                        Real64 MaxSATSuppHeatUser = VRFTU(VRFTUNum).MaxSATFromSuppHeatCoil;
+                        ReportSizingOutput(cZoneHVAC_TU_VRF,
+                                           ZoneHVAC_TU_VRF_Name,
+                                           "Design Size Maximum Supply Air Temperature from Supplemental Heater [C]",
+                                           MaxSATSuppHeatDes,
+                                           "User-Specified Maximum Supply Air Temperature from Supplemental Heater [C]",
+                                           MaxSATSuppHeatUser);
+                        if (DisplayExtraWarnings) {
+                            if (std::abs(MaxSATSuppHeatDes - MaxSATSuppHeatUser) > (4.0 * AutoVsHardSizingDeltaTempThreshold)) {
+                                ShowMessage("SizePTUnit: Potential issue with equipment sizing for " + cZoneHVAC_TU_VRF + ' ' + ZoneHVAC_TU_VRF_Name);
+                                ShowContinueError("User-Specified Maximum Supply Air Temperature from Supplemental Heater of " +
+                                                  RoundSigDigits(MaxSATSuppHeatUser, 2) + " [C]");
+                                ShowContinueError("differs from Design Size Maximum Supply Air Temperature from Supplemental Heater of " +
+                                                  RoundSigDigits(MaxSATSuppHeatDes, 2) + " [C]");
+                                ShowContinueError("This may, or may not, indicate mismatched component sizes.");
+                                ShowContinueError("Verify that the value entered is intended and is consistent with other components.");
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+        if (VRFTU(VRFTUNum).SuppHeatCoilType_Num == DataHVACGlobals::Coil_HeatingWater) {
+            bool ErrorsFound = false;
+            WaterCoils::SetCoilDesFlow(
+                VRFTU(VRFTUNum).SuppHeatCoilType, VRFTU(VRFTUNum).SuppHeatCoilName, VRFTU(VRFTUNum).MaxHeatAirVolFlow, ErrorsFound);
+        }
+
+        if (VRFTU(VRFTUNum).SuppHeatingCoilPresent) {
+            CompType = VRFTU(VRFTUNum).SuppHeatCoilType;
+            CompName = VRFTU(VRFTUNum).SuppHeatCoilName;
+            SizingMethod = DataHVACGlobals::HeatingCapacitySizing;
```

Comment:
Not saying this is wrong since it uses only 1 sizing method (which I like), but there are 2 heating capacity sizing methods. One for water coils (WaterHeatingCapacitySizing) and the other for all other coil types (HeatingCapacitySizing).

Reply:
@rraustad I missed the two heating coil capacity sizing method flavors. Addressed. I will run my test files if it makes a difference.